debug/org.eclipse.core.externaltools/src/org/eclipse/core/externaltools/internal/launchConfigurations/LaunchAttributeArguments.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2025 ArSysOp.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Alexander Fedorov (ArSysOp) - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.core.externaltools.internal.launchConfigurations;
+
+import org.eclipse.core.externaltools.internal.IExternalToolConstants;
+import org.eclipse.core.runtime.preferences.PreferenceMetadata;
+import org.eclipse.debug.core.LaunchAttributeDefined;
+import org.eclipse.debug.core.LaunchAttributeIdentity;
+
+public final class LaunchAttributeArguments implements LaunchAttributeDefined<String> {
```

Comment:
Couldn't we replace this entire class by a static field `LaunchAttributeDefined<String>LAUNCH_ATTRIBUTE_ARGUMENTS` e.g in the `IExternalToolConstants` class? Implemented either with an anonymous implementation for each field or a static factory method (that contains it).
I wonder the same for the companion `LaunchAttributeDefined` implementations.

Reply:
done